### PR TITLE
Visual changes to editor tasks screen.

### PR DESCRIPTION
**Pull request by dbrant**
Changes to the screen itself:
https://phabricator.wikimedia.org/T217167

Changes to onboarding:
https://phabricator.wikimedia.org/T217166

### CODE REVIEW

**Comment by dbrant** [comment]
> Just found that if I have only one Wikipedia language, no matter how many descriptions I've added, the `Translate title descriptions` card is still locked. And the `add languages` card does not show either.

I'm assuming this will be handled by Sharvani's "Update edit feed logic" patch.  In the current master, the "add languages" card doesn't yet exist.

File: app/src/main/java/org/wikipedia/onboarding/OnboardingPageView.java [hunk 1]
```diff
@@ -119,11 +119,19 @@ private void init(@Nullable AttributeSet attrs, @AttrRes int defStyleAttr, @Styl
...
             }
             imageViewCentered.setImageDrawable(centeredImage);
+            if (imageSize > 0 && centeredImage != null && centeredImage.getIntrinsicHeight() > 0) {
+                float aspect = (float)centeredImage.getIntrinsicWidth() / centeredImage.getIntrinsicHeight();
+                LayoutParams params = (LayoutParams) imageViewCentered.getLayoutParams();
+                params.width = (int) imageSize;
+                params.height = (int) (imageSize / aspect);
+                imageViewCentered.setLayoutParams(params);
+            }
             primaryTextView.setText(primaryText);
```
[the remark this answers is not in the document]
Comment (dbrant) [reply app/src/main/java/org/wikipedia/onboarding/OnboardingPageView.java]: done

**Comment by cooltey** [comment]
> > Just found that if I have only one Wikipedia language, no matter how many descriptions I've added, the `Translate title descriptions` card is still locked. And the `add languages` card does not show either.
> 
> I'm assuming this will be handled by Sharvani's "Update edit feed logic" patch. In the current master, the "add languages" card doesn't yet exist.

I see. Thanks.

File: app/src/main/java/org/wikipedia/editactionfeed/EditTasksFragment.java [hunk 2]
```diff
@@ -111,17 +105,16 @@ private void updateRecycler() {
...
+                Prefs.getTotalUserDescriptionsEdited(), Prefs.getTotalUserDescriptionsEdited()));
+        requireActivity().invalidateOptionsMenu();
     }
 
     private void showOneTimeOnboarding() {
-        if (Prefs.showEditTaskOnboarding()) {
+        //if (Prefs.showEditTaskOnboarding()) {
             editOnboardingView.setVisibility(View.VISIBLE);
-        }
+        //}
```
[the remark this answers is not in the document]
Comment (dbrant) [reply app/src/main/java/org/wikipedia/editactionfeed/EditTasksFragment.java]: Damn it! I knew I would forget.

**Pull request merged**